src/utils.ts

```diff
@@ -104,6 +104,12 @@ export function getTSConfig(globals, collectCoverage: boolean = false) {
 
     config.inlineSourceMap = true;
     config.inlineSources = true;
+
+    if (config.outDir) {
```

Comment:
- This `if` isn't really needed.
- Shouldn't this delete should be outside the other `if` as well? As things stand, this will still break coverage reports if a user sets `outDir` in `__TS_CONFIG__`